Model/Adminhtml/Source/PaymentAction.php

```diff
@@ -5,9 +5,6 @@
  */
 namespace CardknoxDevelopment\Cardknox\Model\Adminhtml\Source;
 
-/**
- * Class PaymentAction
- */
```

Comment:
Is there a reason this class does not have a comment?

Reply:
it's not needed. This class description shows below warning, so I removed this lines.
`Class description must contain meaningful information beyond what its name provides or be removed.phpcs
`